app/classifier/classifier.jsx

```diff
@@ -162,7 +165,9 @@ class Classifier extends React.Component {
     }
 
     if (this.props.workflow.configuration.hide_classification_summaries && !this.subjectIsGravitySpyGoldStandard()) {
-      this.props.onCompleteAndLoadAnotherSubject();
+      if (!feedbackActive || (feedbackActive && !isThereFeedback(this.props.subject, this.props.workflow))) {
```

Comment:
This check is different from the later check in the TaskNav component.

Probably not a good idea to have those two different checks in place. I think the code only really works properly if both return the same result, which isn't guaranteed.